### DISCUSSION TITLE
What are these properties on my KeePassXC™ entries created by the KeePassXC™ browser extension?

### DISCUSSION THREAD

**Discussion by ff781** [keepassxreboot/keepassxc#10304] [Q&A]
the key is
KeePassXC-Browser Settings
and the value will be something like
{
"Allow": ["google.com","facebook.com","amazon.com"], "Deny": [], "Realm":""}
}

on a KeePassXC entry for google.com.
So what do facebook and amazon have to do with that google entry?
And what does Allow, Deny, Realm mean?

**Comment by droidmonkey** [comment]
Those control access to entries for specific websites. Not sure why Facebook and Amazon are on that list though. @varjolintu

**Reply by varjolintu** [reply]
Maybe the user has allowed an access to that entry when there has been an iframe with Google login.
